app/assets/javascripts/app.coffee.erb

```diff
@@ -0,0 +1,68 @@
+receta = angular.module('receta',[
+  'templates',
+  'ngRoute',
+  'controllers',
+])
+
+receta.config([ '$routeProvider',
+  ($routeProvider)->
+    $routeProvider
+    .when('/',
+      templateUrl: "<%= asset_path('index.html') %>"
+      controller: 'PcbuilderController'
+    )
+])
+
+components = [
```

Comment:
So this is what you should get out of the API rather than hardcoding it. If we change some naming of categories now (which we did) or possibly add categories we're fucked.